src/cmap/establish/handshake/mod.rs

```diff
@@ -116,19 +112,7 @@ lazy_static! {
                 version: None,
             },
             platform: format!("{} with {}", rustc_version_runtime::version_meta().short_version_string, RUNTIME_NAME),
-        };
-
-        let info = os_info::get();
```

Comment:
This actually performs an `lsb_release` command (if present) and if that fails does some other stuff, so it can take a somewhat significant amount of time to run (~50ms or so in my testing). Given that we only currently use it for the distro version but don't report the distro itself, it seemed worth it to drop for now unti lwe do RUST-1285. (I did this while investigating those timeouts that ultimately ended up being TLS related, but this contributed to it too).

@kmahar are we currently using this version field in a significant way that would inhibit removing it for now?